amundsen_application/static/js/components/ProfilePage/index.tsx

```diff
@@ -10,12 +10,14 @@ import Tabs from 'components/common/Tabs';
 
 import { GlobalState } from 'ducks/rootReducer';
 import { getUserById } from 'ducks/user/reducer';
-import { User, GetUserRequest } from 'ducks/user/types';
+import { GetUserRequest } from 'ducks/user/types';
+
+import { PeopleUser } from 'interfaces';
```

Comment:
`PeopleUser` is a temporary name until we completely reconcile whether or not we can make the switch to `User`.

Reply:
I think it should be `User` as well, however I'm waiting to be sure about what's going on in our backend.

Reply:
I'll leave this for you to clean up as you work on the Profile Page.